### DISCUSSION TITLE
The Events are not showing on previous version it will show all the available events

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5122] [Show and tell]
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/e16024a6-f4c5-4772-843d-2e6c83596a9b)

**Comment by RakulAgn** [comment]
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/a28263a2-3876-444e-8665-7e23980f519f)
This is On Version 0.20.4

**Reply by artf** [reply]
In this version it was not possible to use any other custom event except those defined. I'll check for a fix for the next release
